<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="gui" default="true" />
    <arg name="urdf" default="vlp16_bumblebee" />

    <include file="$(find safer_stack)/launch/spawn/sand_mine.launch">
        <arg name="gui" value="$(arg gui)" />
    </include>

    <include file="$(find safer_stack)/launch/spawn/spawn_husky.launch">
        <arg name="urdf_extras" value="$(find safer_stack)/urdf/$(arg urdf).urdf.xacro" />
    </include>
    

    <rosparam command="load" file="$(find safer_stack)/config/detection.yaml" />

    <node pkg="safer_stack" name="interface_rtabmap" type="interface.py" output="screen">
        <remap from="/edge_distance" to="/rtabmap/edge_distance"/>
        <remap from="/crest" to="/rtabmap/crest"/>
    </node>

</launch>